docs/website/docs/user-guides/v3-migration-guide.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Migrate from v2 to v3
+sidebar_position: 2
```

Comment:
maybe sidebar position 4 so it's below everything else / after the advanced usage guide?